utils/provided-goi-analysis-functions.R

```diff
@@ -292,7 +292,8 @@ plot_goi_expression_umap <- function(normalized_sce,
                         aes(x = X1, y = X2, color = gene_expression)) +
       geom_point(size = 0.01) +
       facet_wrap(as.formula(paste("~", optional_plotting_column))) +
-      scale_color_viridis_c()
+      scale_color_viridis_c() +
+      labs(x = "UMAP1", y = "UMAP2", color = "gene expression")
```

Comment:
```suggestion
      labs(x = "UMAP1", y = "UMAP2", color = "Normalized \ngene expression")
```
Let's try and do this, but make sure the `\n` looks okay and is in the right place. Also can you propagate this suggestion to the rest of the legend labels please.